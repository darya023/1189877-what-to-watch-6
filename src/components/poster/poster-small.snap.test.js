import React from 'react';
import {render} from '@testing-library/react';
import PosterSmall from './poster-small';

it(`Should PosterSmall render correctly`, () => {
  const {container} = render(
      <PosterSmall
        src=""
        alt="Test"
        className="movie-card__poster--small"
      />
  );
  expect(container).toMatchSnapshot();
});
