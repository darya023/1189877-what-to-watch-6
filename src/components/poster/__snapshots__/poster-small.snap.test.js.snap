// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should PosterSmall render correctly 1`] = `
<div>
  <div
    class="movie-card__poster movie-card__poster--small"
  >
    <img
      alt="Test"
      height="327"
      src=""
      width="218"
    />
  </div>
</div>
`;
